src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -205,7 +206,13 @@ private UUID processAudioRequest(Integer requestId) {
             throw new DartsApiException(AudioApiError.FAILED_TO_PROCESS_AUDIO_REQUEST, e);
         }
 
-        mediaRequestService.updateAudioRequestStatus(mediaRequestEntity.getId(), COMPLETED);
+        AudioRequestOutputFormat audioRequestOutputFormat = AudioRequestOutputFormat.MP3;
+        if (mediaRequestEntity.getRequestType().equals(DOWNLOAD)) {
+            audioRequestOutputFormat = AudioRequestOutputFormat.ZIP;
+        }
+        String fileName = mediaRequestEntity.getHearing().getCourtCase().getCaseNumber() + "_" + mediaRequestEntity.getHearing().getHearingDate();
+
+        mediaRequestService.updateAudioRequestCompleted(requestId, fileName, audioRequestOutputFormat);
```

Comment:
The added code could throw an exception, e.g. if the `hearing` does not have a `courthouse` assigned, or if there's a hibernate connectivity issue. Could you move the code into the try block (L166-L192)?

The background - we need to be careful with exception handling in this method (`processAudioRequest`). When the method starts, the `AudioRequestStatus` is updated to `PROCESSING`. If an uncaught exception is thrown, the processing will quit, and the media request record in the DB will remain in the `PROCESSING` state. This will perpetually retried by KEDA, blocking processing of any other media request. As such we need to ensure all exceptions are caught in this method and handled by setting `AudioRequestStatus` to `FAILED`.

Reply:
Moved code inside try block. Moved near start so if it fails it will fail early.